lib/source-map-consumer.js

```diff
@@ -842,8 +842,10 @@ class IndexedSourceMapConsumer extends SourceMapConsumer {
           return cmp;
         }
 
+        // The needle column is 0-based, but the section offset column is
+        // stored 1-based.
         return (
-          aNeedle.generatedColumn - section.generatedOffset.generatedColumn
+          aNeedle.generatedColumn + 1 - section.generatedOffset.generatedColumn
```

Comment:
```suggestion
          aNeedle.generatedColumn - (section.generatedOffset.generatedColumn - 1)
```

Reply:
Since according the the spec, the columns are mostly 0-based and in this case it is the section offset column was stored as 1-based 
for VLQ encoding compatibility. Lets do the adjustment on the section offset column rather than on the generated column. WDYT?